gzip.go

```diff
@@ -21,16 +21,20 @@ const (
 
 type codings map[string]float64
 
-// The default qvalue to assign to an encoding if no explicit qvalue is set.
+// DefaultQVALUE is the default qvalue to assign to an encoding if no explicit qvalue is set.
 // This is actually kind of ambiguous in RFC 2616, so hopefully it's correct.
 // The examples seem to indicate that it is.
-const DEFAULT_QVALUE = 1.0
+const DefaultQVALUE = 1.0
 
 // gzipWriterPools stores a sync.Pool for each compression level for reuse of
 // gzip.Writers. Use poolIndex to covert a compression level to an index into
 // gzipWriterPools.
 var gzipWriterPools [gzip.BestCompression - gzip.BestSpeed + 2]*sync.Pool
 
+// minSize gives the hability to skeep compression on response smaller than the
+// given size. The default value is 512 bytes.
+var minSize = 512
```

Comment:
This is a reasonable value, but the default should be a const here, and the value itself should be a member of GzipResponseWriter. It's awkward to change, but I don't think many people would want to anyway.